contrib/ray/test.sh

```diff
@@ -30,17 +30,27 @@ function trap_handler {
 
   # Delete KubeRay operator
   kustomize build kuberay-operator/base | kubectl -n $NAMESPACE delete -f -
+
 }
 
 trap trap_handler EXIT
 
+kubectl get namespaces --selector=istio-injection=enabled
```

Comment:
kubeflow-user-example-com is the namespace. It must be a proper multi-tenancy supported kubeflow profile namespace